lib/rspec-puppet-facts.rb

```diff
@@ -255,11 +254,20 @@ def self.common_facts
     return @common_facts if @common_facts
 
     @common_facts = {
-      puppetversion: Puppet.version,
+      # from Facter itself
```

Comment:
what do you mean by that comment? `RbConfig::CONFIG['sitelibdir']` isn't provided by facter?